### ISSUE TITLE
Dynamic ToolList

### ISSUE THREAD

**Issue by phibla** [umati/umatiGateway#116]
At the moment we are implementing an OPC UA Server with dynamic tool list. In the specification the following is mentioned: 

`The contents of the ToolListType instance can change during the Server runtime (e.g. if tools are inserted into the machine tool or removed from it). A change in the list can be indicated using the NodeVersion Property and the GeneralModelChangeEventType. The NodeVersion and the GeneralModelChangeEventType are intended to be used in the way defined in OPC 10000-3 and 7.3.
`
Is the Dashboard-OPCUA-Client already able to handle this behaviour?

**Comment by ccvca** [comment]
No, the client does not hanlde `GeneralModelChangeEvents` at the moment.

**Comment by mdornaus** [comment]
Allready implemented for umati/v2.
